src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+export const Person = ({ person }) => {
```

Comment:
You could destructure instead of creating a variable in the body of the component
```
export const Person = ({ person: { name, age, // etc} }) => {
```